cli/gitaskpass_test.go

```diff
@@ -30,38 +30,41 @@ func TestGitAskpass(t *testing.T) {
 		}))
 		t.Cleanup(srv.Close)
 		url := srv.URL
-		cmd, _ := clitest.New(t, "--agent-url", url, "Username for 'https://github.com':")
+		inv, _ := clitest.New(t, "--agent-url", url, "Username for 'https://github.com':")
+		inv.Environ.Set("GIT_PREFIX", "/")
```

Comment:
It's weird that this is called `Environ` instead of `Env`. All of the OS functions are `SetEnv`

Reply:
See https://pkg.go.dev/os#Environ

Environ is used as the plural form.